low-level/files/opt/seagate/sspl/conf/sspl.config.tmpl.1-node

```diff
@@ -0,0 +1,45 @@
+cortx:
+  release:
+    product: LR2
+    setup: cortx
+server_node:
+  TMPL_MACHINE_ID:
+    bmc:
+      ip: TMPL_BMC_IP
```

Comment:
You need to talk in the wiki what this value should be in case of VM environment.

Reply:
Okay.